File: commands/advice.lua
local vkapi = require "./api/vkapi"
local httpreq = require "./api/httpreq"

return {
    keywords = {"совет"};
    process = function(data)
        local answ = httpreq.req.GETJSON("http://fucking-great-advice.ru/api/random")
        vkapi.message.send(data.peer_id, answ.text)
    end
}

File: commands/advice.js
const vk = require('../api/vk')
const axios = require('axios')

module.exports = {
    keywords: ['совет'],
    callback: async (data) => {
        const response = await axios.get('http://fucking-great-advice.ru/api/random')
        vk.message.send(data.peer_id, response.data.text)
    }
}